dojson/contrib/to_marc21/model.py

```diff
@@ -50,5 +52,15 @@ def do(self, blob):
 
         return output
 
+    def _get_indicators(self, value):
+        """Get indicators from the value."""
+        if isinstance(value, MutableMapping):
+            return value.pop('$ind1', '_'), value.pop('$ind2', '_')
+        if isinstance(value, MutableSequence):
+            return [(v.pop('$ind1', '_'), v.pop('$ind2', '_'))
+                    for v in value][0]
```

Comment:
I need to remove the `$ind1` and `$ind2` from every single dictionary inside the list.
This is how they look:

```python
value = [
    {'$ind1': '1', '$ind2': '2', 'a': 'foo', 'b': 'bar'},
    {'$ind1': '1', '$ind2': '2', 'a': 'foo', 'b': 'bar'},
    {'$ind1': '1', '$ind2': '2', 'a': 'foo', 'b': 'bar'}
]
```
And this is what I need:

```python
ind1 = 1
ind2 = 2
value = [
    {'a': 'foo', 'b': 'bar'},
    {'a': 'foo', 'b': 'bar'},
    {'a': 'foo', 'b': 'bar'},
]
```